tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -65,14 +81,17 @@ spec:
             exit 1
         fi
 
-        floatingTag=$(jq -r '.images.floatingTag // ""' $DATA_FILE)
+        floatingTagsCount=$(jq -rc '.images.floatingTags | length' $DATA_FILE)
```

Comment:
`| length` will return just the number, so both `-r` and `-c` are redundant here. (`-r` only makes sense for strings and `-c` only makes a difference if the result is a json, not just a single value.)